google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/ProjectDebuggeeBinding.java

```diff
@@ -199,7 +199,7 @@ public void run() {
             }
           });
 
-          LOG.error("Error listing debuggees from Cloud Debugger API", ex);
+          LOG.warn("Error listing debuggees from Cloud Debugger API", ex);
```

Comment:
What's the exact message?  To fix #396 we need to let the user know the
difference between a project with no debuggable modules, and a project for
which they don't have debug access.

On Mon, Feb 1, 2016 at 6:17 PM, akerekes <notifications@github.com> wrote:

> In
> google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/ProjectDebuggeeBinding.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/404#discussion_r51497796>
> :
>
> > @@ -199,7 +199,7 @@ public void run() {
> >              }
> >            });
> >
> > -          LOG.error("Error listing debuggees from Cloud Debugger API", ex);
> > +          LOG.warn("Error listing debuggees from Cloud Debugger API", ex);
>
> The user sees the message to select another project with debuggable module
> in the dialog.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/404/files#r51497796>
> .
>